src/components/TweetItem/TweetItem.tsx

```diff
@@ -1,32 +1,43 @@
 import * as React from 'react';
+import { inject, observer } from 'mobx-react';
 import { faComment, faHeart } from '@fortawesome/free-regular-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 
 import './TweetItem.scss';
 
-import { ITweet } from '../../interfaces';
+import { ITweet, IStores } from '../../interfaces';
 
 export interface ITweetItemProps {
   tweet: ITweet,
+  stores?: IStores
 }
 
+@inject("stores")
+@observer
 export class TweetItem extends React.Component<ITweetItemProps, {}> {
   constructor(props: ITweetItemProps) {
     super(props);
+    this.handleRemoveTweet = this.handleRemoveTweet.bind(this);
```

Comment:
the same, 
in declaring methods use
`methodName = () => {......}` instead - so you won't be needed to use`.bind(this)` in controller